physical/raft/fsm.go

```diff
@@ -642,6 +653,17 @@ func (f *FSM) ApplyBatch(logs []*raft.Log) []interface{} {
 						err = b.Put([]byte(op.Key), op.Value)
 					case deleteOp:
 						err = b.Delete([]byte(op.Key))
+					case getOp:
+						val := b.Get([]byte(op.Key))
+						if len(val) > 0 {
```

Comment:
Don't we want to append an entrySlice even when the len is 0?

Reply:
If I'm following you here, this is to make sure we have the same number of entries in the slice as we have get ops? And then I guess at the other end, we just verify the value is present before updating anything.

Reply:
Fixed in https://github.com/hashicorp/vault/pull/17151/commits/a5a96152575cf331726bb5b50de29bb9fcb65712